lib/balanced_vrp_clustering.rb

```diff
@@ -191,7 +191,7 @@ def move_capacity_violating_dataitems
             end
           end
         end
-        # puts "#{needs_moving_up} \tpoints needs love, #{@moved_down} of them moved_down, #{@moved_up} of them moved_up, #{needs_moving_up - @moved_down - @moved_up} of them untouched \tviolations=#{@clusters_with_capacity_violation.collect.with_index{ |array, i| array.empty? ? ' _ ' : "|#{i + 1}|" }.join(' ')}"
+        # puts "#{@needed_moving} \tpoints needs love, #{@moved_down} of them moved_down, #{@moved_up} of them moved_up, #{@needed_moving - @moved_down - @moved_up} of them untouched \tviolations=#{@clusters_with_capacity_violation.collect.with_index{ |array, i| array.empty? ? ' _ ' : "|#{i + 1}|" }.join(' ')}"
```

Comment:
```suggestion
```

Reply:
maybe we should find a way to have a debug mode ? or an independent commit with debug logs